trade-finance-manager-ui/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
This ensures the `--legacy-peer-deps` flag is always applied to relevant npm commands in this project.